cluster-autoscaler/utils/drain/drain.go

```diff
@@ -31,9 +31,7 @@ import (
 const (
 	// PodDeletionTimeout - time after which a pod to be deleted is not included in the list of pods for drain.
 	PodDeletionTimeout = 12 * time.Minute
-)
 
-const (
```

Comment:
This way we're mixing two distinct constant types together - I'd rather keep them separate.

Reply:
Reverted